translations/en_US.json

```
@@ -14,11 +14,13 @@
   "landing_hero_description_no_extension": "Looks like you don’t have the <span class=\"paragraph-inter-16-semi-bold\">Universal Profile browser extension</span> installed. You’ll need it before you can log in to our wallet service.",
   "error_no_accounts": "Account not found.",
   "asset_all_creators_unverified": "Un-verified creators",
+  "not_up_description": "This does not seem to be a Universal Profile address. View this address on Blockscout to find out more information. ",
```

```suggestion
  "not_up_description": "This does not seem to be a Universal Profile address. View this address on Blockscout to find out more information.",
```

not sure if intended but there is an extra space at the end

---

Ah right, I'll remove